documentation/decisions/adr/000N-video-conferencing.md

```diff
@@ -0,0 +1,166 @@
+
+- **Technical Challenges**: Technical issues, such as poor internet connectivity, audio/video glitches, and compatibility problems, can disrupt meetings and lead to frustration.
+- **Security Concerns**: Video conferencing tools may pose security risks, like unauthorized access, data breaches, or potential privacy violations if not properly configured and managed.
+- **Fatigue and Burnout**: Frequent video meetings can lead to "Zoom fatigue" or virtual meeting burnout, affecting participant engagement and focus.
+- **Distractions and Multitasking**: Participants may be prone to distractions or multitasking during virtual meetings, reducing attention and active engagement.
+- **Learning Curve**: New users may find it challenging to navigate and fully utilize the features of the video conference tool, leading to a learning curve for some team members.
+- **Bandwidth Consumption**: Video conferencing consumes significant internet bandwidth, which may impact other online activities if the network is not robust enough. This could also be limiting for individuals with low or slow broadband connection.
+
+### Back-up Options
+
+
+
+## Comparison Matrix
+
+- ✅ Feature available, meets requirement
+- ❌ Feature not available, does not meet requirement
+- 🔄 Partial feature, limited feature availability, feature in progress or undergoing improvements
+- 1-3 Strength level
+- ❓Unknown
+
+| Factor                      | Zoom | Microsoft Teams | Google Meet | Jitsu |
+| --------------------------- | :--------: | :--------: | :---------: | :-----: | 
+| Public Access                   |     ✅      |   ✅    |      ✅      |    ✅    |  
+| Waiting Room               |    ✅     |  ✅   |     ✅      |   ✅    |   
+| Video Recording              |    ✅     |   ✅   |     ✅      |   ✅    |  
+| Screen Sharing             |     ✅     |  ✅   |     ✅      |   ✅    |  
+| Chat                |     ✅     |   ✅   |     ✅      |   ✅    |  
+| Phone Support                        |    ✅     |  ✅   |     ✅      |   ✅    |   
+| Live Transcription               |     ✅     |   ✅   |     ✅      |    ❌   |   
+| Authority to Operate  |     x      |   ✅    |      ?      |    ?    |  
+| Webinar* |     ✅      |   ✅    |      3      |    🔄    |  
+| Breakout Rooms*        |     ✅     |   ✅   |     ✅      |   🔄    |  
+| Open Source*      |    ❌     |   ❌   |     ❌     |   ✅   |  
+| Attendance Tracking*                 |    ✅     |  ✅   |     🔄      |   ❓    |  
+| Community & Support*         |    3     |   3   |     3      |   3   |  
+| Scalability*                 |     ✅     |   ✅   |     ✅      |   ✅    |  
+
+
+* Nice to have
+
+## Pros and cons of each option <!-- OPTIONAL -->
+
+### Zoom
+
+Zoom is a widely used video conferencing platform known for its ease of use and comprehensive features. It offers virtual meetings, webinars, breakout rooms, screen sharing, chat functionality, and phone support. 
+
+#### Additional details and pricing
+
+- **Pricing**: 
```

Comment:
Zoom seems like the most commonly and easily used in my humble experience, but I've heard it's pricey for enterprise use? Idk, without thinking about it too much, it would probably get my (uninformed) vote unless the costs are prohibitive.

Reply:
Ended up selecting Zoom and Jitsu as a backup option. Zoom can be expensive with Enterprise but we can limit the number of paid users. The public can join for free and it allows us to have the enterprise features (like Webinars which will be helpful for the community management piece of it)